targets/XPlatCppSdk/source/code/include/playfab/PlayFabWinHttpPlugin.h

```diff
@@ -43,4 +43,4 @@ namespace PlayFab
         std::deque<std::unique_ptr<CallRequestContainerBase>> pendingRequests;
         std::deque<std::unique_ptr<CallRequestContainerBase>> pendingResults;
     };
-}
\ No newline at end of file
+}
```

Comment:
extra space ?

Reply:
It's a force of habit of mine to have a single blank space at the end of a file due to some legacy languages expecting this.

I tried removing the change yesterday, but git isn't counting it as a change for the commit. I think I'll have to change the file directly in github.

---
In reply to: [261333906](https://github.com/PlayFab/SDKGenerator/pull/419#discussion_r261333906) [](ancestors = 261333906)